boa/src/builtins/value/operations.rs

```diff
@@ -18,6 +18,13 @@ impl Value {
             return number::equals(f64::from(self), f64::from(other));
         }
 
+        //Null has to be handled specially because "typeof null" returns object and if we managed
+        //this without a special case we would compare self and other as if they were actually
+        //objects which unfortunately fails
+        if self.is_null() {
+            return true;
+        }
+
```

Comment:
Can we add a test for this, and a link to the spec?

Reply:
Isn't boa/src/builtins/value/tests.rs:58 already a test for this?

Reply:
Yes! Perfect, rhanks! :)